recipe/build.sh

```diff
@@ -40,7 +40,9 @@ make install PREFIX="${PREFIX}"
 # As OpenBLAS, now will have all symbols that BLAS or LAPACK have,
 # create libraries with the standard names that are linked back to
 # OpenBLAS. This will make it easier for packages that are looking for them.
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/libblas.a
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/liblapack.a
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/libblas$SHLIB_EXT
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/liblapack$SHLIB_EXT
+
```

Comment:
I guess we can trim this blank line as the comment above still seems to apply to this.